fava/application.py

```diff
@@ -429,7 +429,10 @@ def jump():
     qs_dict = url.decode_query()
     for key, values in request.args.lists():
         if len(values) == 1 and values[0] == "":
```

Comment:
`and key in qs_dict` might be better than relying on exceptions, since this is not an _exceptional_ case :smile: